background.ts

```diff
@@ -346,6 +352,22 @@ Electron.ipcMain.on('k8s-integration-set', async(event, name: string, newState:
   }
 });
 
+/** The first time this event is called the data arrives somewhat slowly.
+ *  Shuffle the names so the order varies each time (not by much with only 3 items,
```

Comment:
Mainly for aesthetic reasons.

The reasons didn't appear immediately, and they didn't appear in the same order on the page, top to bottom.

I thought there were two alternatives:
1. Hardwire the actual order on the page, and load them that way. It adds extra brittle coupling, but OTOH the order isn't that likely to change.
2. Randomize it.

Since this brought up issues, I'll hardwire the order.

Reply:
If we had some simple sort, to the user it's still a (non-obvious at least) order?